ckanapi/cli/main.py

```diff
@@ -107,9 +107,9 @@ def main(running_with_paster=False):
     if arguments['action']:
         try:
             for r in action(ckan, arguments):
-                sys.stdout.write(r)
+                sys.stdout.write(str(r))
```

Comment:
this might not work. if the output is pretty json and includes non-ascii characters then we would need `stdout = getattr(sys.stdout, 'buffer', sys.stdout)` and no `str(r)` here